python/graphscope/framework/graph.py

```diff
@@ -156,6 +164,14 @@ def _close_learning_instances(self):
             instance.close()
         self._learning_instance_list.clear()
 
+    def _launch_interactive_instance_impl(self):
+        try:
+            sess = get_session_by_id(self.session_id)
+            sess.gremlin(self)
+        except:  # noqa: E722
+            # suppress all exceptions
+            pass
```

Comment:
Record the exception, and when failed, rethrow when users call `.gremlin()`.

Reply:
Record error msg in `InteractiveQuery` when launching failed.